wasm/section.go

```diff
@@ -10,11 +10,19 @@ import (
 	"fmt"
 	"io"
 	"io/ioutil"
+	"sort"
 
 	"github.com/go-interpreter/wagon/wasm/internal/readpos"
 	"github.com/go-interpreter/wagon/wasm/leb128"
 )
 
+type sectionInf interface {
```

Comment:
not super fond of this interface name.
but I haven't a better name to propose, perhaps except `sectionType` ?

in hindsight, `Section` should have probably been named `RawSection` and `sectionInf` named `Section`...